docs/index.rst

```diff
@@ -79,18 +96,51 @@ Supported Devices
   - TL-WR710N (v1)
   - TL-WR740N (v1, v3, v4)
   - TL-WR741N/ND (v1, v2, v4)
+  - TL-WR743N/ND (v1, v2)
   - TL-WR841N/ND (v3, v5, v7, v8, v9)
   - TL-WR842N/ND (v1, v2)
-  - TL-WR941N/ND (v2, v3, v4)
+  - TL-WR941N/ND (v2, v3, v4, v5)
+  - TL-WR2543N/ND (v1)
 
 * Ubiquiti
 
   - Bullet M2
   - Nanostation M2
+  - Nanostation M XW
+  - Loco M XW
   - Picostation M2
   - UniFi AP
+  - UniFi AP Pro
   - UniFi AP Outdoor
 
+ar71xx-nand
+-----------
+
+* Netgear
+  - WNDR3700 (v4)
+  - WNDR3700 (v1)
+
+mpc85xx-generic
+---------------
+
+* TP-Link
+  - TL-WDR4900 (v1)
+
+ramips-rt305x
+-------------
```

Comment:
`ramips-rt305x` is BROKEN so far as it doesn't support AP+Adhoc, it shouldn't be mentioned in the list of officially supported models.